### ISSUE TITLE
Mediapipe

### ISSUE THREAD

**Issue by AndreAhmed** [DavidBoja/SMPL-Anthropometry#23]
Dear @DavidBoja 
Thanks so much for your contributions. 
Is there a way to use Mediapipe, to use it for your measurement SMPL anthropometry ? If that's possible what needs to be changed and how ?

**Comment by DavidBoja** [comment]
As far as I know, Mediapipe works on images of humans, and finds various features from these, like joints, face detection, etc.?

You can use Mediapipe to estimate the joints from a subject in the image, and then use an existing HMR method to estimate the SMPL from the image. Then, you can use the SMPL-Anthropometry to get the measurements.

**Comment by AndreAhmed** [comment]
@DavidBoja But as you mentioned before I need a T or A pose.. this won't work ?

**Comment by DavidBoja** [comment]
Yes, you can just use the shape parameters that you obtain with your HMR method and initialize the Measurer with that. This will measure an SMPL in the T-pose with the desired shape.
The A-pose should also be fine because there is not much variation from the T-pose to the A-pose in terms of measurements (as they are estimated in this repo).

If you want to measure posed subjects, you can check out our other work on [pose-independent anthropometry](https://github.com/DavidBoja/pose-independent-anthropometry).

**Comment by AndreAhmed** [comment]
@DavidBoja Thanks so much for the reference github.
The main problem with all that work, Is I want to be able to view the measurement in Unity3D ( A game engine). Not sure how would that happen

**Comment by DavidBoja** [comment]
Can't really help you with Unity3D as I've never used it unfortunately.
